project.ruleset.xml.example

```diff
@@ -10,25 +11,75 @@
 	<exclude-pattern>/docroot/wp-content/plugins/*</exclude-pattern>
 	<exclude-pattern>*.twig</exclude-pattern>
 
-	<rule ref="Squiz.PHP.CommentedOutCode"/>
-	<rule ref="Squiz.WhiteSpace.SuperfluousWhitespace"/>
-	<rule ref="Generic.CodeAnalysis.UnusedFunctionParameter"/>
-	<rule ref="Generic.Commenting.Todo"/>
-	<rule ref="Generic.ControlStructures.InlineControlStructure"/>
+	<!-- Exclude Composer Vendor directory. -->
+	<exclude-pattern>/vendor/*</exclude-pattern>
 
 	<!--
-	We may also want to to include all the rules in a standard
+	We may also want to to include all the rules in a standard.
 	-->
-	<rule ref="WordPress-Core">
+	<rule ref="WordPress-Extra">
 		<!--
 		We may want a middle ground though. The best way to do this is add the
-		entire ruleset, then rule by rule, remove ones that don't suit a project. We
-		can do this by running `phpcs` with the '-s' flag, to see the names of the
-		different Sniffs, as their rules are broken. From here, we can opt to
+		entire ruleset, then rule by rule, remove ones that don't suit a project.
+		We can do this by running `phpcs` with the '-s' flag, to see the names of
+		the different Sniffs, as their rules are broken. From here, we can opt to
 		exclude problematic sniffs like so.
 		-->
 
 		<exclude name="WordPress.WhiteSpace.ControlStructureSpacing" />
 		<exclude name="WordPress.XSS.EscapeOutput" />
 	</rule>
+
+	<!-- Let's also check that everything is properly documented. -->
+	<rule ref="WordPress-Docs">
```

Comment:
Thanks for catching this. I'm clearly working on too many different things at the same time ;-)